jimpass/managers/bitwarden.py

```diff
@@ -66,7 +66,7 @@ def get_session(self) -> str:
         """ Get the key holding the session hash """
         code, stdout = srun("keyctl request user bw_session")
         if code != 0 or not stdout:
-            code, passwd = rofi(
+            code, passwd = prun(
```

Comment:
Hm... this is a problem. Only `Controller` and `Bitwarden` know about the config, hence runner.
A quick fix would be to pass the `runner` as an argument to the `BitwardenSession` constructor as well, over in `Bitwarden.__init__`.
But the cleaner option IMO would be to just put these methods into `Bitwarden`. I don't see the value of this class anyway.